Versions/Wrath/bestride.lua

```diff
@@ -1,11 +1,8 @@
 
 function BeStride:Version_OnEnable()
-    BeStride_Constants.spells.druid.travelform = 783
     BeStride_Constants.spells.druid.flightform = 33943
     if IsPlayerSpell(40120) then
         BeStride_Constants.spells.druid.flightform = 40120
-    else
-        BeStride_Constants.spells.druid.flightform = 33943
```

Comment:
We should maybe change this to swiftflightform and add a flightform spellid as well with the option to have both available and usable.

Reply:
What I did is to set the flightform spellid by default and overwrite it with swiftflightform spell id when this spell is known. I think it's okay like that, as the first flight form is not available to player anymore when he have learn the swift version.